notification/notification.go

```diff
@@ -67,12 +67,12 @@ func NewTeamInvitationEmail(userID string, teamName string, inviterName string,
 // inviter - the name of the user sending the invitation
 // roleNames - a comma-separated list of role names
 // acceptToken - the unique acceptance token value
-func NewSpaceInvitationEmail(userID string, spaceName string, inviterName string, roleNames string, acceptURL string) Message {
+func NewSpaceInvitationEmail(identityID string, spaceName string, inviterName string, roleNames string, acceptURL string) Message {
 	return Message{
 		MessageID:   uuid.NewV4(),
 		MessageType: "invitation.space.noorg",
-		TargetID:    "",
-		UserID:      &userID,
+		TargetID:    identityID,
```

Comment:
Good catch, could you please add a sample payload somewhere in the PR description which works?